PopupMenu/PopupMenu.js

```diff
@@ -43,6 +43,13 @@ const PopupMenuBase = kind({
 		onClose: PropTypes.func,
 		onHide: PropTypes.func,
 		open: PropTypes.bool,
+		/**
+		 * The layout orientation of the component
+		 *
+		 * @type {('horizontal')}
+		 * @default 'horizontal'
+		 * @private
+		 */
```

Comment:
As i checked the docs file, this part has not been exposed before.
So there is no need to fix this.

Reply:
We have to fix it becaus there is no docs code (or comments) in PopupMenu So,I think someday someone add comments for PopupMenu for documetation.